### ISSUE TITLE
Accessibility - Document roles and responsible parties on the DS site

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#5221]
There has been some confusion about what people get out of the box when it come to accessibility when teams use tokens vs grommet + hpe-theme vs ds site guidance.

We should create some documentation on the design system site that we can point people towards that answers these questions.

This documentation should be pretty lightweight and easy for people to scan and get an understanding of the different levels of support. In addition we should highlight what areas are the DS team's responsibility vs what areas need to be handled by the designer/developer.

Thinking this documentation could live here: https://design-system.hpe.design/foundation/accessibility?q=access#built-into-all-design-system-tools

There is some good info in slides 29 and 30 of this presentation that we may be able to leverage. https://hpe.sharepoint.com/:p:/r/teams/hpe-design/_layouts/15/Doc.aspx?sourcedoc=%7BA4D4BFEB-4E53-4E1C-8493-CB93B0F737D7%7D&file=HPE.DS.Adoption.Scorecard.Update_July03-NextSteps.pptx&action=edit&mobileredirect=true

**Comment by jcfilben** [comment]
Linking Matt's Gemini chat https://gemini.google.com/share/510c7c9b732b

**Comment by KennyAtHPE** [comment]
[Current draft](https://www.figma.com/design/YXoHxIOWdBnih4UCh70fS2/Accessibility-Documentation?node-id=0-1&t=5VxZ01uifLxY9ajJ-1)
